scss/bitstyles/tools/_directional-properties.scss

```diff
@@ -122,3 +122,21 @@
     }
   }
 }
+
+@mixin output-block(
+  $object-level,
+  $classname-root,
+  $breakpoint-suffix: ''
+) {
+  $classname: get-classname(
+    (
+      $bitstyles-namespace,
+      $object-level,
+      $classname-root,
+    )
+  );
+
+  .#{$classname}#{$breakpoint-suffix} {
+    @content;
+  }
+}
```

Comment:
This seems to be pretty useful :) 
I wonder why it is only used here... 
Do you want to introduce this in other places as well?

Reply:
Yes exactly — I’ve already added generators for the directional utility classes (margin-right/-left etc), and the non-directional (e.g. grid-cols), this is the last one(?), that outputs a block of CSS in a standardised classname. Now that this exists, I would start using it throughout so we can provide every class at different breakpoints in a consistent manner